docs/user/rstudio/ide/guide/ui/ui-panes.qmd

```diff
@@ -2,54 +2,54 @@
 title: "Pane Layout"
 ---
 
-The RStudio User interface is broken up into 4 primary panes:
+The RStudio user interface has 4 primary panes:
 
-1.  **Source pane**\
-2.  **Console pane**\
-3.  **Environment pane**, containing the **Environment**, **History**, **Connections**, **Build**, and **VCS** tabs\
-4.  **Output pane**, containing the **Files**, **Plots**, **Packages**, **Help**, **Tutorial**, **Viewer**, and **Presentation** tabs
+-  **Source pane**\
+-  **Console pane**\
+-  **Environment pane**, containing the **Environment**, **History**, **Connections**, **Build**, and **VCS** tabs\
+-  **Output pane**, containing the **Files**, **Plots**, **Packages**, **Help**, **Tutorial**, **Viewer**, and **Presentation** tabs
```

Comment:
The Tutorial tab seems to have been moved to the Environment pane. I checked SW 2022.07.2 and on `main` branch. I'm not sure when this happened.

![image](https://user-images.githubusercontent.com/9591545/191776045-d0e5727a-dd7c-4920-b0d9-385b509d993b.png)

Reply:
The tabs in these two panes are interchangeable depending on what the user selects in the Pane Layout options here:
<img width="599" alt="image" src="https://user-images.githubusercontent.com/5323711/191779602-cb015403-1b04-4c96-96a1-402ca2e0f257.png">

Reply:
Oof - I think you're right! I thought I had gone back to all defaults... (I miss my dark mode theme 😢)

I do also agree with Melissa that this is customizable, but I can update docs to reference Tutorial as default under Environment pane.

Reply:
Changed